Using data disk has issues with data not persisted

nguada opened the pull request:
•	Scenario: 
o	VMSS using managed data disk is having issues with data not being persisted
•	Repro:
o	Customer has created a VMSS  with managed disks using a gallery image CentOS.
o	Customer then followed the article https://docs.microsoft.com/en-us/azure/virtual-machine-scale-sets/virtual-machine-scale-sets-attached-disks to attached a data disk to VMSS.
o	Data disk he attached was an empty data disk.
o	Now he goes and updates this disk  on the VM instance in the VMSS and then mounts this data disk (instance _0)
o	Later customer tried to put some data on the disk  by logging in to the instance. 
o	Customer realized when a new instance is created, that data disk is coming empty.

azurecla commented:
Hi __@v-kagri__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-kagri). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

nguada committed:
Using data disk has issues with data not persisted

•	Scenario: 
o	VMSS using managed data disk is having issues with data not being persisted
•	Repro:
o	Customer has created a VMSS  with managed disks using a gallery image CentOS.
o	Customer then followed the article https://docs.microsoft.com/en-us/azure/virtual-machine-scale-sets/virtual-machine-scale-sets-attached-disks to attached a data disk to VMSS.
o	Data disk he attached was an empty data disk.
o	Now he goes and updates this disk  on the VM instance in the VMSS and then mounts this data disk (instance _0)
o	Later customer tried to put some data on the disk  by logging in to the instance. 
o	Customer realized when a new instance is created, that data disk is coming empty.

nguada committed:
Using data disk has issues with data not persisted

Applying new format to documentation for article based on feedback received. (#2620)

nguada committed:
Misspell word at the title
